packages/react-components/react-nav-preview/library/src/components/SplitNavItem/useSplitNavItem.tsx

```diff
@@ -23,12 +25,15 @@ export const useSplitNavItem_unstable = (
 
   const { size = 'medium' } = useNavContext_unstable();
 
+  const { value: potentialParenValue } = useNavCategoryContext_unstable();
+
+  const mainNavItemType = potentialParenValue ? 'navSubItem' : 'navItem';
```

Comment:
I think this could be a useful boolean (isSubNav?) that could be added to state for additional context for users if they are implementing the hooks or need that info for additional styling etc.

Reply:
updated :-)